libnetwork/controller.go

```diff
@@ -871,11 +872,14 @@ func (c *Controller) NetworkByID(id string) (*Network, error) {
 }
 
 // NewSandbox creates a new sandbox for containerID.
-func (c *Controller) NewSandbox(containerID string, options ...SandboxOption) (_ *Sandbox, retErr error) {
+func (c *Controller) NewSandbox(ctx context.Context, containerID string, options ...SandboxOption) (_ *Sandbox, retErr error) {
 	if containerID == "" {
 		return nil, types.InvalidParameterErrorf("invalid container ID")
 	}
 
+	_, span := otel.Tracer("").Start(ctx, "libnetwork.Controller.NewSandbox")
```

Comment:
This is an _active_ bug. Multiple calls below are being passed the wrong context, leading to incorrect traces.
```suggestion
	ctx, span := otel.Tracer("").Start(ctx, "libnetwork.Controller.NewSandbox")
```